mlir/include/mlir/Dialect/MIOpen/utility/common.h

```diff
@@ -7,4 +7,7 @@ struct ArgumentFields {
   StringRef gemmTargetCharName[3];
```

Comment:
Just a random side comment, I actually didn't know we have a `common.h` that get pulled in by both:
 - LowerMIOpenOps.h
 - BackwardWeightV4R4Helper.h

Those seem to only deal with a particular lowering pass and is not that "common". I think when time is right we should work on refactoring to split the `LowerMIOpenOps.h` to per pass based. Then get rid of those helper/common headers.